api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -181,6 +181,49 @@
         return getOptionalValue(propertyName, arrayType).map(Arrays::asList);
     }
 
+    /**
+     * Return the resolved configuration properties type with the specified prefix. 
+     * The type declaration can be annotated with 
+     * {@linkplain org.eclipse.microprofile.config.inject.ConfigProperties @ConfigProperties}. 
+     * If the type is annotated with {@linkplain org.eclipse.microprofile.config.inject.ConfigProperties @ConfigProperties}, 
+     * the prefix supplied in this method overrides the prefix associated with the annotation 
+     * {@linkplain org.eclipse.microprofile.config.inject.ConfigProperties @ConfigProperties}.
+     * 
+     * @param <T> 
+     *              The Class Type
```

Comment:
At the top of the documentation, you say "Return the resolved configuration properties type" (which is of type `T`).  Here you call it a "Class Type".  Which is it?

Reply:
The top means an instance and T is class. I can reword to 
`Return the resolved configuration properties instance.` Thoughts? If you have a better suggestions, please file here.

Reply:
updated to instance, same to the other overloaded `getConfigProperties(Class, String)`